airflow/utils/db.py

```diff
@@ -55,6 +55,9 @@
 
 # We need to add this model manually to get reset working well
 from airflow.models.serialized_dag import SerializedDagModel  # noqa: F401
+
+# Alias this to WebSession so it doesn't conflict with sqla.orm.session.Session
+from airflow.models.session import Session as WebSession  # noqa: F401
```

Comment:
maybe we should name the model itself WebSession so as to not conflict?

Reply:
No longer an issue 🎉